html/includes/print-menubar.php

```diff
@@ -639,7 +639,7 @@
 
 if ($_SESSION['authenticated']) {
     echo('
-           <li><a href="logout/"><i class="fa fa-sign-out fa-fw fa-lg" aria-hidden="true"></i> Logout</a></li>');
+           <li><a href="' . $config['auth_logout_handler']  . '"><i class="fa fa-sign-out fa-fw fa-lg" aria-hidden="true"></i> Logout</a></li>');
```

Comment:
Should switch to `Config::get()` here as well.

Reply:
Done, by changing how this is done. The template isn't really the right place to handle authentication stuff as it's prone to being altered. I implemented the same functionality by making the URL the user is sent to after log out configurable.